File: Timus_Problems/Volume_One/a_plus_b.cpp
//
// Author: James Ortiz
// File: a_plus_b.cpp 
// Description: Calculate a+b
//

#include <iostream>
using namespace std;

long long int summation(long long int a, long long int b){
    return a + b;
}

int main(void){
    long long int a, b;

    cin >> a >> b;
    
    long long int sum = summation(a, b);

    cout << sum;

    return 0;
}

File: Timus_Problems/Volume_One/a_plus_b.py
#
# Author: James Ortiz
# File: a_plus_b.py
# Description: Calculate a+b (Number: 1000)
# Input: 1 5
# Output: 6
#

def sum(a, b):
    return a + b

a, b = map(int, input().split())
print(sum(a, b))

